crates/rolldown/tests/esbuild/dce/remove_unused_pure_comment_calls/artifacts.snap

```diff
@@ -8,7 +8,7 @@ source: crates/rolldown_testing/src/integration_test.rs
 ```js
 
 //#region entry.js
-function bar() {}
+// @__PURE__ // @__PURE__ // @__PURE__ // @__PURE__ // #__PURE__ // #__PURE__ // #__PURE__ // #__PURE__ function bar() {}
```

Comment:
Oh no!

Reply:
Let me follow this, I think I know what happened here.